### PR TITLE
repo setup

### PR DESCRIPTION
**Pull request by shiprashipra**
Updated folder and readme

### CODE REVIEW

File: src/test/resources/application.conf [hunk 1]
```diff
@@ -15,8 +15,28 @@
...
     host: "http://localhost"
+    auth {
+      port = 8585
+      productionRoute = ""
+    }
+    ims {
+      port = 9653
+      productionRoute = ""
+    }
+    tgp {
```
Comment (mauro-fanton) [inline src/test/resources/application.conf]: is this the trader-goods-profiles API?

Comment (Nidhi1shah) [reply src/test/resources/application.conf]: @shiprashipra You can use this ports - https://github.com/hmrc/trader-goods-profiles/blob/main/conf/application.conf

is that correct @mauro-fanton ?

Comment (shiprashipra) [reply src/test/resources/application.conf]: Please review it again updated it

Comment (Nidhi1shah) [reply src/test/resources/application.conf]: Thanks. Looks fine to me. @mauro-fanton are you happy with the changes?

Comment (mauro-fanton) [reply src/test/resources/application.conf]: just would like to know what service id TGP, if this is the trader-goods-profile API then the port number is wrong

**Review by mauro-fanton — APPROVED** [review]

**Pull request merged**